code/style.css

```diff
@@ -1,4 +1,273 @@
-html {
-  background: #0025ff;
-  color: #fff;
+body {
+  font-family:Verdana, Geneva, Tahoma, sans-serif;
+  box-sizing: border-box;
+  margin: 0;
+  background-color: whitesmoke;
 }
+
+header {
+  font-family: 'Rampart One', cursive;
+  display: flex;
+  background-color: #ffff4d;
+  justify-content: center;
+  width: 100%; 
+  height: 20vh;
+}
+
+.header-text {
+  font-size: 7vw;
+  align-self: center;
+}
+
+.head-logo {
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  margin-top: 30px;
+}
+
+.site-logo {
+  width: 70px;
+  height: 70px;
+  object-fit: cover;
+  border-radius: 50%;
+  display: none;
+  margin-top: 15px;
+}
+
+nav {
+  display: flex;
+  justify-content: center;
+  padding: 5px;
+  background-color: #ffff4d;
+  margin-bottom: 15px;
+}
+ 
+.nav-menu {
+  padding: 5px;
+  color: black;
+}
+
+a:hover {
+  color: white;
+  font-size: large;
+
+}
+article > h1 {
+  color: black;
```

Comment:
I really enjoyed the colorfulness of the "Event" and "Music" boxes. A suggestion for the text color would be to use a darker grey to make the text easier to read on some of the colored boxes.